schemas/component/component.schema

```diff
@@ -9,6 +9,48 @@
         },
         {
             "properties": {
+                "$policies": {
+                    "type": "array",
+                    "items": { 
+                        "oneOf": [
+                            {
+                                "type": "object",
+                                "properties": {
+                                    "type": {
+                                        "type": "string", 
+                                        "const": "requiresAction"
```

Comment:
>requiresAction [](start = 50, length = 14)

This should either be requires or requiresKinds or kinds should become actions.

Reply:
changed to requiresKind